xpdacq/beamtime.py

```diff
@@ -168,6 +168,65 @@ def Tramp(dets, exposure, Tstart, Tstop, Tstep, *, md=None):
     yield from plan
 
 
+def T_list(dets, exposure, T_list):
+    """
+    Scan over the temperature controller in a list of user defined steps
+
+    Parameters
+    ----------
+    dets : list
+        list of objects represent instrument devices. In xpdAcq, it is
+        default to area detector.
+    exposure : float
+        total time of exposure in seconds for area detector
+    T_list : list
+        a list of temperature steps that you wish to scan over
```

Comment:
is it a list of temperature steps, or a list of temperatures where a scan is to be run?